simapp/types/types.go

```diff
@@ -0,0 +1,34 @@
+/*
+Package types defines the interfaces and types required for an SDK-based app.
```

Comment:
Is this the only file in this package? Why not have it under `simapp/types.go`?

Reply:
it would generate dep cycles. I can move it to `/types` if you want